packages/wasi/src/index.ts

```diff
@@ -1,3 +1,1405 @@
-import WASI from "./wasi";
+/* eslint-disable no-unused-vars */
```

Comment:
Did you make any changes here? Or was it a simple copy over? 😄

Reply:
Simple copy over with two line changes for the default exports.